### PR TITLE
feat: add LSMASHWorks

### LINKED ISSUE #34: Mr-Ojii build版L-SMASH Works
## 名前

L-SMASH Works

## 概要

https://github.com/Mr-Ojii/L-SMASH-Works-Auto-Builds

にて**毎日**最新のffmpegを用いてビルドがされている。これにはMr-Ojiiがパッチを当てたbranchのものを含む。

## 説明

すでにPOPn氏のビルドのものがあるが、その後のffmepgの更新に追従できているとは言えない。

## 開発者

muken, HolyWu, Mr-Ojii氏

## 紹介ページURL
 
https://github.com/Mr-Ojii/L-SMASH-Works

## ダウンロードページURL

https://github.com/Mr-Ojii/L-SMASH-Works-Auto-Builds/releases

## ファイル名

see release

# 懸念点

- 毎日ビルドが回るのでそれに追従するCIをこちらで組んだ場合commit logがそれで埋まる
    - ffmepgの更新に毎日追従する必要はあるのか
        - その場合xmlのデータ構造に手を加えるなどしてcommit数を圧縮することが必要ではないか
            - 参考事例: AURでは`-git` sufixがついたパッケージが一般的にある: https://aur.archlinux.org/packages/ffmpeg-git/
    - 一方である一定期間過ぎたら取り込まれているべきではないか

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Add multiple LSMASHWorks.

**This PR depends on the following PR(s).**
- https://github.com/hal-shu-sato/apm/pull/216
- #39

## Related Issue

Closes #34

## How Has This Been Tested?

- Install and uninstall the package without any problems.

## Checklist:

- [x] Updated the modification date in `mod.xml`.

### CODE REVIEW

File: data/mod.xml [hunk 1]
```diff
@@ -2,5 +2,5 @@
 <?xml-model href="../schema/mod.xsd" type="application/xml" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <mod>
 	<core>2021-06-20T12:05:00+09:00</core>
-	<packages_list>2021-11-14T09:50:00+09:00</packages_list>
+  <packages_list>2021-11-14T13:00:00+09:00</packages_list>
```
Comment (hal-shu-sato) [inline data/mod.xml:5]: It's very trivial, but the indent should be a tab, but it's a space.

**Review by hal-shu-sato — COMMENTED** [review]
There is a very trivial thing.

**Comment by mitosagi** [comment]
It is now clean.

**Review by hal-shu-sato — APPROVED** [review]
Thanks for fixing it.

**Pull request merged**